acceptancetests/assess_persistent_storage.py

```diff
@@ -223,6 +222,63 @@ def assess_charm_removal_single_block_and_filesystem_storage(client):
     else:
         log.info('Detach single block device storage output check: {}'.format(
             detach_single_blk_output_check))
+    # storage status change after remove-application takes some time.
+    # from experiments even 30 seconds is not enough.
+    time.sleep(60)
```

Comment:
this should be a wait_for method. Feel free to pool against status or something similar as needed until the right status shows up. A random sleep time will fail when run on different machines / substrates.